src/components/Header/Header.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {
+  FormEvent,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import callError from '../../utils/callError';
+import { addTodo, USER_ID } from '../../api/todos';
+import { MainContext } from '../../ContextProvider/ContextProvider';
+
+const Header: React.FC = ({}) => {
+  const context = useContext(MainContext);
+  const { todos, setTodos, setError, loadingIds, setLoadingIds, setTempTodo } =
+    context;
+
+  const [todoInputValue, setTodoInputValue] = useState('');
+  const [isDisabled, setIsDisabled] = useState(false);
+
+  const focusedInput = useRef(null);
```

Comment:
```suggestion
  const inputRef = useRef(null);
```
It's ref with input as a value. It's not always focused